### ISSUE TITLE
syntax error in change_url not recognized?

### ISSUE THREAD

**Issue by jedie** [YunoHost/package_linter#90]
https://github.com/YunoHost-Apps/pyinventory_ynh/commit/63a8914139aef01a18283b13b3e4aa1b0f31f776#diff-8b4bf6fd453b044a4960771a329a995e177d9b895918c3548831eb039badfd55R38 has a syntax error:

```
redis_db=$(ynh_app_setting_get --app="$app" --key=redis_db
```
Forgotten ")" at the end of the line.

But the linter passes, see: https://github.com/YunoHost-Apps/pyinventory_ynh/runs/1565074491

**Comment by alexAubin** [comment]
Eeeeeh yeah that's a bit weird that shlex (or i forgot the name of the lib we're using to parse bash) doesn't complain about it ... On the other hand the main goal is more to report for bad practices (and also good ones, sometimes :sweat_smile:) than bash syntax issues :s
